frontend/src/global_styles/content/user-content/_typography.sass

```diff
@@ -43,6 +43,7 @@
   margin: 0
   line-height: 1.6em
   color: inherit
+  word-break: break-all
```

Comment:
@Kharonus this line causes https://community.openproject.org/projects/openproject/work_packages/44125/activity?query_id=3470 since it's applied to all paragraphs now.

Do you know where this originated from, and can we apply it to modal only?